test/suite.go

```
@@ -65,18 +73,51 @@
-	s.Require().NoError(err)
-	s.NotEqual(types.Hash{}, stateRoot)
-	s.Greater(maxBytes, uint64(0))
+	genesisTime, lastStateRoot, _ := s.initChain(ctx, uint64(1))
+
+	for i, c := range cases {
+		s.Run(c.name, func() {
+			err := s.TxInjector.InjectTxs(c.txs)
+			s.Require().NoError(err)
+			txs, _ := s.Exec.GetTxs(ctx)
+			fmt.Println(len(txs))
+			stateRoot, maxBytes, err := s.Exec.ExecuteTxs(ctx, c.txs, uint64(i+1), genesisTime.Add(time.Duration(i)*time.Second), lastStateRoot)
```

golangci-lint (1.62.2)

[high] G115: integer overflow conversion int -> uint64

(gosec)